openvdb_wolfram/OpenVDBLink/Source/GlueTensors.h

```diff
@@ -0,0 +1,420 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+#ifndef OPENVDBLINK_GLUETENSORS_HAS_BEEN_INCLUDED
+#define OPENVDBLINK_GLUETENSORS_HAS_BEEN_INCLUDED
+
+#include "OpenVDBCommon.h"
+#include "LTemplate.h"
+
+#include <openvdb/openvdb.h>
+
+#include <tbb/parallel_for.h>
+#include <tbb/parallel_reduce.h>
+
+#include <vector>
+
+
+// ------------ Scalar classes ------------ //
```

Comment:
Please consider enclosing this in a namespace.

Reply:
This should be resolved now. They are now under the namespace `openvdbmma::glue`.